docs/architecture/adr-004-nakamoto-bonus.md

```diff
@@ -0,0 +1,166 @@
+
+$$
+rd_{ki} = \frac {d_{ki}}{x_{ji}} * r_{ji}.
+$$
+
+
+Where:
+
+- $rd_{ki}$ is the reward obtained by delegator $k$ for block $i$.
+- $d_{ki}$ is the stake delegate by delegator $k$ at block $i$.
+- $x_{ji}$ is the stake of validator $j$ at block $i$.
+- $r_{ji}$ is the reward of validator $j$ for block $i$.
+
+We can define a new metric, `Reward Per Stake (RPS)` as the amount a delegator would be rewarded for unit of stake, and we can compute it as:
+
+$$
+RPS_{ji} = \frac{r_{ji}}{x_{ji}}  
+$$
+
+Where:
+
+- $RPS_{ji}$ is the reward per stake for validator $j$ for block $i$.
+- $r_{ji}$ is the reward of validator $j$ for block $i$.
+- $x_{ji}$ is the stake of validator $j$ at block $i$.
+
+In essence, RPS tells us how much a delegator would receive by delegating to validator $j$. 
+
+In the current model, it can easily be seen that RPS is equivalent across all validators and it can be derived from the first equation of this ADR that is equal to:
+
+$$
+RPS = \frac{R_{i}}{S{i}}
+$$
+
+This means, that from the point of view of a delegator, the decision on which validator to choose has no influence on their expected reward. Therefore, they might be as well delegating their stake to the most established validators.
+
+## The Nakamoto Bonus
+
+In this ADR, we propose to split the reward in two components, proportional reward (PR) and Nakamoto Bonus (NB) such that:
+
+$$
+R_i = PR_i + NB_i.
+$$
+
+Where:
+- $R_i$ is the total reward obtained by validating block $i$.
+- $PR_i$ is the proportional reward for block $i$.
+- $NB_i$ is the Nakamoto Bonus for block $i$.
+
+Moreover, we define a new parameter $\eta : 0 \le eta \le 1$ that specifies how NB is computed from the reward: $ NB_i = R_i * \eta $.
+Increasing $\eta$ increases NB, and decreasing it, decreases NB.
+
+The reward obtained by a validator $j$ at block $i$ is computed as:
+
+$$
+r_{ji} = \frac {x_{ji}}{S_i} * PR_i + \frac{NB_i}{N_i}
+$$
+
+Where:
+- $r_{ji}$ is the reward of validator $j$ for block $i$.
+- $x_{ji}$ is the stake of validator $j$ at block $i$.
+- $PR_i$ is the proportional reward for block $i$.
+- $S_i$ is the total stake across all validators at block $i$.
+- $NB_i$ is the Nakamoto Bonus for block $i$.
+- $N_i$ is the total number of validators for block $i$.
+
+
+The way the reward obtained by a validator is distributed across delegators remains unchanged.
+
+If we compute the RPS, as defined before:
+
+$$
+RPS_{ji} = \frac{r_{ji}}{x_{ji}}  
+$$
+
+It can be easily seen that it is not possible anymore to remove the dependency on the validator choosen as it was without the Nakamoto Bonus.
+
+More specifically, 
+
+
+$$
+RPS_{ji} = \frac{r_{ji}}{x_{ji}} = \frac{PR_i}{S_i} + \frac{NB_i}{N_i * x_{ji}} 
+$$
+
+It can be seen that, the first member of the sum of RPS_{ji} is independent from $j$, the specific validator, while the second is inversly proportional to x_{ji} - the stake of validator $j$ at block $i$.
```

Comment:
```suggestion
It can be seen that, the first member of the sum of $RPS_{ji}$ is independent from $j$, the specific validator, while the second is inversly proportional to $x_{ji}$ - the stake of validator $j$ at block $i$.
```